java/src/main/java/com/bazaarvoice/gumshoe/Context.java

```diff
@@ -25,6 +25,7 @@
     public static enum Status {
         CREATED, STARTED, FINISHED;
     }
+    private Gumshoe gumshoe;
```

Comment:
Oh, what I meant is line 36, there's another reference to a Gumshoe named governor that looks to be unused?